CalibCalorimetry/EcalLaserCorrection/python/ecalLaserCorrectionService_cfi.py

```diff
@@ -1,5 +1,9 @@
 import FWCore.ParameterSet.Config as cms
 
-EcalLaserCorrectionService = cms.ESProducer("EcalLaserCorrectionService")
+EcalLaserCorrectionService = cms.ESProducer("EcalLaserCorrectionService", 
+                                        ### delta t is the safety margin before stopping
+                                        ### the laser correction extrapolation (this is in seconds)
+                                        maxExtrapolationTimeInSec = cms.uint32( 0 ) )
+
 
```

Comment:
Hi,

Now with fillDescriptions, you no longer need the explicit cfi file in the ../python directory. Upon compilation of the c++ code with scram, a cfi file will be generated from fillDescriptions and placed into ../cfipython, accessible thru the same path  CalibCalorimetry.EcalLaserCorrection.EcalLaserCorrectionService_cfi